buzzmobile/constants.yaml

```diff
@@ -11,7 +11,7 @@ pixels_per_m: 80
 # Used for tentacle projection, travel distance is the step between each 
 # tentacle point, and num_points is the length of each tentacle, if using
 # liner, and not tentacle projection, ignore
-travel_distance: 0.3 
+travel_distance: 0.061
```

Comment:
So this travel distance is merely how far out to extend the tentacles. If they're too long, we will be caring about things too far into the future (and therefore make bad predictions).

For my initial tests, I saw that 0.3 worked well. However, since then the image frame's dimensions and the value of `pixels_per_meter` have changed. So this commit makes `travel_distance` somewhat consistent with those.